packages/api/prisma/schema.prisma

```diff
@@ -91,6 +93,26 @@ model ProjectSummary {
   @@map("projectSummary")
 }
 
+model Vote {
+  id               Int      @id @default(autoincrement())
```

Comment:
It would be useful to have a table with voting strategies and an entry for QFVote
And a vote would have a link to that VotingStrategy entry